### ISSUE TITLE
Compatibility with mst-async-task

### ISSUE THREAD

**Issue by 10000multiplier** [ConrabOpto/mst-query#15]
Can I safely use `mst-query` with [mst-async-task](https://github.com/jetako/mst-async-task)?

Thanks for creating a great library.

**Comment by k-ode** [comment]
Yes, it should work. However, you cannot replace `run` on a query or mutation with a mst-async-task.

**Comment by k-ode** [comment]
It's still experimental, but you can now use queries and mutations as properties just like tasks in mst-async-task.

```ts
const UserStore = types
    .model({
        getUser: types.optional(GetUserQuery, {}),
        updateUser: types.optional(UpdateUserMutation, {}),
    })
    .actions(self => ({
        afterCreate()  {
             self.getUser.run();
        }
    }):
```
It's available in mst-query@1.1.0-alpha.3
